downloadCurrentVersion.ts

```diff
@@ -3,73 +3,27 @@
 // This is used to download the correct binary version
 // as part of the prepublish step.
 
-import * as fs from 'fs';
-import { https } from 'follow-redirects';
-import MemoryStream from 'memorystream';
-import { keccak256 } from 'js-sha3';
+import downloader from './downloader';
 const pkg = require('./package.json');
+const DEFAULT_HOST = 'https://binaries.soliditylang.org';
 
-function getVersionList (cb) {
-  console.log('Retrieving available version list...');
+async function download (version, host = DEFAULT_HOST) {
+  try {
+    const list = JSON.parse(await downloader.getVersionList(host));
+    const releaseFileName = list.releases[version];
+    const expectedFile = list.builds.filter(function (entry) { return entry.path === releaseFileName; })[0];
```

Comment:
I'm assuming `list.builds` is an array since you're calling `filter` on it - would it make sense to use `find` instead?

Reply:
Yes. I replaced it.